app/assets/javascripts/miq_application.js

```diff
@@ -1456,8 +1458,14 @@ function miqAccordionSwap(_collapse, expand) {
 
 // This function is called in miqOnLoad
 function miqInitToolbars() {
-  $("#toolbar button:not(.dropdown-toggle), #toolbar ul.dropdown-menu > li > a, #toolbar .toolbar-pf-view-selector > ul.list-inline > li > a").off('click');
-  $("#toolbar button:not(.dropdown-toggle), #toolbar ul.dropdown-menu > li > a, #toolbar .toolbar-pf-view-selector > ul.list-inline > li > a").click(miqToolbarOnClick);
+  $("#toolbar:not(.miq-toolbar-menu) button:not(.dropdown-toggle), "+
+  "#toolbar:not(.miq-toolbar-menu) ul.dropdown-menu > li > a, "+
+  "#toolbar:not(.miq-toolbar-menu) .toolbar-pf-view-selector > ul.list-inline > li > a"
+  ).off('click');
+  $("#toolbar:not(.miq-toolbar-menu) button:not(.dropdown-toggle), " +
+  "#toolbar:not(.miq-toolbar-menu) ul.dropdown-menu > li > a, "+
+  "#toolbar:not(.miq-toolbar-menu) .toolbar-pf-view-selector > ul.list-inline > li > a"
+  ).click(miqToolbarOnClick);
```

Comment:
Right, well, you're changing this anyway so why not :)  
(But I'll need to do a global PR for these anyway, since jquery 3 removes a bunch of those in favor of explicit `on`..)